application/run-tests.sh

```diff
@@ -1,15 +1,13 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
Perhaps a comment refreshing people's memory about the functionality of the set -e and set -x commands would be helpful. Lest they want to put the rc=$? code back in.